data-prepper-plugins/blocking-buffer/src/main/java/org/opensearch/dataprepper/plugins/buffer/blockingbuffer/BlockingBuffer.java

```diff
@@ -43,8 +43,8 @@
 @DataPrepperPlugin(name = "bounded_blocking", pluginType = Buffer.class)
 public class BlockingBuffer<T extends Record<?>> extends AbstractBuffer<T> {
     private static final Logger LOG = LoggerFactory.getLogger(BlockingBuffer.class);
-    private static final int DEFAULT_BUFFER_CAPACITY = 512;
-    private static final int DEFAULT_BATCH_SIZE = 8;
+    private static final int DEFAULT_BUFFER_CAPACITY = 12_800;
```

Comment:
Seems like a reasonable default. I am pretty sure in most cases an even higher buffer capacity will have better performance.